entity-framework/core/modeling/relationships.md

```diff
@@ -188,7 +188,7 @@ You can use the string overload of `HasForeignKey(...)` to configure a shadow pr
 
 #### Foreign key constraint name
 
-By convention, when targeting a relational database, foreign key constraints are named FK_<dependent type name>_<principal type name>_<foreign key property name>. For composite foreign keys <foreign key property name> becomes an underscore separated list of foreign key property names.
+By convention, when targeting a relational database, foreign key constraints are named FK_\<dependent type name>_\<principal type name>_\<foreign key property name>. For composite foreign keys \<foreign key property name> becomes an underscore separated list of foreign key property names.
```

Comment:
The _ is being parsed as italics.

Reply:
Maybe just fence it?

Reply:
fine either way you fix it. Just fix it. 😆